invenio_previewer/ext.py

```diff
@@ -76,6 +76,8 @@ def css_bundles(self):
 
     @property
     def js_bundles(self):
+        if self.app.config['PREVIEWER_ASSETS_USE_WEBPACK']:
+            return self.app.config['PREVIEWER_BASE_JS_BUNDLES'] + ['vendor.js']
```

Comment:
Just a comment, should you add `vendor.js` here instead, where also you have the manifest?
https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/templates/invenio_previewer/base.html#L35